azkaban-common/src/main/java/azkaban/server/SlaRequestUtils.java

```diff
@@ -85,8 +93,42 @@ public static List<SlaOption> parseSlaOptions(final HttpServletRequest req, fina
     return slaOptions;
   }
 
-  private static SlaOption parseSlaSetting(final String set, final String flowName, final List<String> emails,
-      final Map<String, Map<String, String>> alertersConfigs) throws ServletException {
+  private static Map<String, Map<String, List<String>>> getStringListMapFromStringMap(
```

Comment:
Might want to consider adding a test for this in the future, as that might clarify what is happening a little :)

Reply:
Added a comment in the meantime, hopefully it helps to clarify what the method does, thanks!